dali/operators/random/uniform_distribution.h

```diff
@@ -0,0 +1,173 @@
+// Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
```

Comment:
```suggestion
// Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
```

Reply:
I actually wrote it in 2020, but OK, changed